test/wallet-account.test.js

```diff
@@ -2,6 +2,8 @@ const url = require('url');
 const localStorage = require('localstorage-memory');
 const BN = require('bn.js');
 
+const MULTISIG_HAS_METHOD = 'add_request_and_confirm';
```

Comment:
It would be helpful to understand where the value is coming from: 
maybe reference the const from wallet-account.ts or leave a comment that this is where this value is defined?

Reply:
added similar comment from wallet-account.ts to this test